lib/app/endpoint/terminology.rb

```diff
@@ -0,0 +1,224 @@
+# frozen_string_literal: true
+
+require_relative '../utils/terminology'
+require 'sinatra/custom_logger'
+
+module Inferno
+  class App
+    class Terminology < Endpoint
+      set :prefix, '/fhir'
+      Inferno::Terminology.register_umls_db('umls.db')
+      Inferno::Terminology.load_valuesets_from_directory('resources', true)
+      set :logger, Logger.new('terminology_misses.log')
```

Comment:
We could do that, yeah. I wanted this to log to a separate log file to keep them separated, and because I expect this logger to go away once we've been able to add more code systems and valuesets.